docs/server/source/server-reference/configuration.md

```diff
@@ -319,7 +319,8 @@ holding the logging configuration.
         "granular_levels": {
             "bichaindb.backend": "info",
             "bichaindb.core": "info"
-        }
+        },
+        "port": 7070
```

Comment:
Please add the name of the associated environment variable to the big list at the top of this page as well. I guess it would be `BIGCHAINDB_LOG_PORT` 

Add it right after `BIGCHAINDB_LOG_GRANULAR_LEVELS`